.github/workflows/prettify.yml

```diff
@@ -0,0 +1,27 @@
+name: Auto-format with Prettier
+on:
+  push:
+    branches: ['development']
+  pull_request:
+    branches: ['development']
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16.13.0'
+      - name: Install and build dependencies
+        run: yarn
+      - name: Run prettier
+        run: yarn prettier-write
```

Comment:
We run prettier --check as our linter step in the ci.yml.  Would it make sense to remove that from the linter now? Would it make since to just make this a step in the ci.yml?

Reply:
Good question… I guess it makes sense to stop using it in our CI jobs, since we're fixing it anyway 🤔